internal/github/profile_test.go

```
@@ -0,0 +1,333 @@
+	})
+
+	svr := httptest.NewServer(router)
+	defer svr.Close()
+
+	// generate valid key for testing
+	key := generateKey(t)
+
+	// Example of an invalid profile URL
+	configURL, err := url.Parse("github.com/chinmina/chinmina-bridge/docs/profile.yaml")
+	if err != nil {
+		fmt.Errorf("url conversion from string format failed: %w", err)
```

golangci-lint (1.64.8)

Error return value of `fmt.Errorf` is not checked

(errcheck)